### PR TITLE
Some more fixes to book-edited.tex

### PR DESCRIPTION
**Pull request by chrishtr**
Two of these changes were in the previous copy editing PR but didn't make it into book-edited.tex.

...and one change to invalidation.md to match.

### CODE REVIEW

File: book-edited.tex [hunk 1]
```diff
@@ -19516,7 +19516,7 @@ \section{Overlap and Transforms}\label{overlap-and-transforms}}
 
 However, if you try to load the example above, you'll find that it still
 looks wrong---the blue square is supposed to be \emph{under} the green
-one, but it's on top.\footnote{Hit-testing is correct, though, because
+one, but it's on top.\footnote{Hit testing is correct, though, because
```
Comment (chrishtr) [inline book-edited.tex:19519]: It seems we settled on "hit testing", no "hit-testing", though the copy editor seems
inconsistent about it.

File: book-edited.tex [hunk 2]
```diff
@@ -21247,7 +21247,7 @@ \section{Keyboard Navigation}\label{keyboard-navigation}}
 \texttt{Tab}s. Note that hitting \texttt{Enter} when focused on a text
 entry clears the text entry; in most browsers, it submits the containing
 form instead. That quirk is a workaround for our browser
-{not implementing} the
+not implementing the
```
Comment (chrishtr) [inline book-edited.tex:21250]: Unnecessary ref?

File: book-edited.tex (hunk 1 above)
Comment (pavpanchekha) [reply book-edited.tex:19519]: Can we check if we really are consistent? No hyphen seems right, we are testing for a hit.

File: book-edited.tex (hunk 2 above)
Comment (pavpanchekha) [reply book-edited.tex:21250]: It’s how internal links end up compiler

File: book-edited.tex [hunk 3]
```diff
@@ -5930,7 +5930,7 @@ \section{Outline}\label{LayingOutPages-outline}}
 \section{Exercises}\label{LayingOutPages-exercises}}
 \begin{enumerate}[label=\thechapter-\arabic*]
 \item \emph{Links bar.} At the top and bottom of the web version of
-  each chapter of this book there is
+  each chapter of this book is
```
Comment (pavpanchekha) [inline book-edited.tex:5933]: I prefer the “there”, was the copy editor against?

File: book-edited.tex [hunk 4]
```diff
@@ -11822,7 +11822,7 @@ \section{A Login System}\label{a-login-system}}
   The server will display who added which guest book entry.
 \end{itemize}
 
-Let's start coding. We'll hard-code two user/password pairs:
+Let's start coding. We'll hard-code two username--password pairs:
```
Comment (pavpanchekha) [inline book-edited.tex:11825]: Ooh, good find

File: book-edited.tex [hunk 5]
```diff
@@ -14796,8 +14796,8 @@ \section{Optimizing Surface Use}\label{optimizing-surface-use}}
 \end{bookblock}
 \footnotetext{
 There is also
-  \href{https://en.wikipedia.org/wiki/Computer_Graphics:_Principles_and_Practice}{Computer
-  Graphics: Principles and Practice}, which incidentally I remember
+  \href{https://en.wikipedia.org/wiki/Computer_Graphics:_Principles_and_Practice}{\emph{Computer
+  Graphics: Principles and Practice}}, which incidentally I remember
```
Comment (pavpanchekha) [inline book-edited.tex:14800]: Technically you want textit here, not emph.

**Review by pavpanchekha — APPROVED** [review]

File: book-edited.tex (hunk 3 above)
Comment (chrishtr) [reply book-edited.tex:5933]: It wasn't in their copy.

File: book-edited.tex (hunk 1 above)
Comment (chrishtr) [reply book-edited.tex:19519]: I checked.

File: book-edited.tex (hunk 2 above)
Comment (chrishtr) [reply book-edited.tex:21250]: Right. I guess if doesn't change the output it's fine.

**Pull request merged**